tests/html/bluetooth_device_disconnect.html

```diff
@@ -95,6 +91,11 @@
     function log(line) {
         document.getElementById("log").textContent += line + '\n';
     }
+
+    function sleepFor( sleepDuration ){
```

Comment:
Let's say it is mostly acceptable on real-time systems. The setTimeout is better (as Matyas said in our encrypted language :D )